examples/Chat/Main.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE PackageImports    #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+import           "base" Control.Applicative
+import           "base" Control.Monad.IO.Class              (liftIO)
+import           "concur-core" Concur.Core.Types            (Widget)
+import           "concur-replica" Concur.Replica            (HTML, runDefault)
+import           "concur-replica" Concur.Replica.DOM        (button, input, li, text, ul)
+import           "concur-replica" Concur.Replica.DOM.Events (BaseEvent (target), onClick, onInput, targetValue)
+import           "concur-replica" Concur.Replica.DOM.Props  (value)
```

Comment:
Hm, doesn't seem to compile with the explicit `concur-replica` package import:

```
    Could not find module ‘Concur.Replica.DOM.Events’
    It is not a module in the current program, or in any known package.
   |
11 | import           "concur-replica" Concur.Replica.DOM.Events (BaseEvent (target), onClick, onInput, targetValue)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Seem to do fine without any `PackageImports`, could you remove them?